platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -114,6 +116,7 @@ public void run() {
     }).start();
   }
 
+  @UiThread
```

Comment:
I thought this was a requirement as the manager itself must be loaded from main thread afaik, let me fix this up.